src/lib/clone-object.ts

```diff
@@ -1,3 +1,7 @@
 export default function cloneObject<T>(obj: T): T {
+  if (obj === undefined) {
```

Comment:
```suggestion
  if (typeof obj === 'undefined') {
```

Reply:
An alternative option! Not sure if there are pros/cons

Reply:
nah this works too and is a little cleaner